README.md

```diff
@@ -1,19 +1,38 @@
-| [Google Season of Docs](https://developers.google.com/season-of-docs)             | Mentoring initiative for the technical writers                                                                                            | [seasonofdocs](https://github.com/cncf/mentoring/blob/main/programs/seasonofdocs#readme)     |
-| [Outreachy](https://www.outreachy.org)                                            | Mentoring initiative for the communities traditionally underrepresented in tech                                                           | [outreachy](https://github.com/cncf/mentoring/blob/main/programs/outreachy#readme)           |
-| [CROSS](https://cross.ucsc.edu/programs/osre2021.html)                            | The CROSS Research Experiences program provides support for undergraduate and graduate students contributing to CROSS incubator projects. | [CROSS](https://github.com/cncf/mentoring/blob/main/programs/archive/cross#readme)                   |
-
+| [LFX Mentorship (ex-CommunityBridge)](https://mentorship.lfx.linuxfoundation.org) | Mentoring initiative by the Linux Foundation                                                                                              | [lfx-mentorship](/programs/lfx-mentorship#readme) |
+| [Google Summer of Code](https://summerofcode.withgoogle.com/)                     | Mentoring program for the students                                                                                                        | [summerofcode](/programs/summerofcode#readme)     |
+| [Google Season of Docs](https://developers.google.com/season-of-docs)             | Mentoring initiative for the technical writers                                                                                            | [seasonofdocs](/programs/seasonofdocs#readme)     |
+| [Outreachy](https://www.outreachy.org)                                            | Mentoring initiative for the communities traditionally underrepresented in tech                                                           | [outreachy](/programs/outreachy#readme)           |
+| [CROSS](https://cross.ucsc.edu/programs/)                            | The CROSS Research Experiences program provides support for undergraduate and graduate students contributing to CROSS incubator projects. | [CROSS](/programs/archive/cross#readme)                   |
 
 
 ## Code of Conduct
 
-CNCF Mentoring programs follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md).
+Please note that CNCF Mentoring programs follow the [CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md). By participating in this project you agree to abide by its terms.
+
+# Thank you for participating in the CNCF Mentoring Program!
```

Comment:
IMO this looks a bit off when I preview the page.

Reply:
But, perhaps it was intentional to make it very big :)

Reply:
Agreed, ment to do a second level heading ## :smile: 
Updated!